### ISSUE TITLE
Is it possible to put the spec_augmenter in front of the nn.BatchNorm2d

### ISSUE THREAD

**Issue by garrywrj** [qiuqiangkong/audioset_tagging_cnn#40]
Hi,
SpecAugmentation masks a block of consecutive time steps or mel frequency channels. But why the order is input, BatchNorm, spec_augmenter? Is there any reason for it. Can i adjust the order to input, spec_augmenter, BatchNorm?

Thanks

**Comment by qiuqiangkong** [comment]
Because log mel has large negative values. It may be better to apply spec_augment on the normalized data.

**Comment by garrywrj** [comment]
SpecAugmentation is directly applied to the input log mel spectrogram in google's original paper. But I'm not sure is there any Batchnormalization layer after SpecAugmentation due to my lack of asr knowledge. Is there any experiment or paper to supoort your point

**Comment by qiuqiangkong** [comment]
Thanks! We have not experimented the comparision yet.

**Comment by garrywrj** [comment]
Thanks for your quick reply.
